src/endpoints/Endpoint.js

```diff
@@ -121,17 +105,20 @@ export default class Endpoint {
           let error
           let value
           try {
-            value = this.jsonToAny(json, options)
+            value = this.jsonToAny(json, stateOptions)
           } catch (e) {
             error = e.message
+            console.error(error)
```

Comment:
I'm not sure if we should expose technical information. It makes debugging much easier when customers send us screenshots

Reply:
Hmm I'm also not sure. But I think since this is JS Client code, which is (generally) visible to the user (even though it's pretty ugly to work with once bundled) and therefore it may be alright, if we expose technical information on what's going wrong in the client side code.

Reply:
Yeah exposing it ok. I'll leave it as it is because it the obfuscated error message still helps more than some generic text.